g_docs.go

```diff
@@ -770,6 +770,30 @@ func parseObject(d *ast.Object, k string, m *swagger.Schema, realTypes *[]string
 
 				var tagValues []string
 				stag := reflect.StructTag(strings.Trim(field.Tag.Value, "`"))
+				
+				defaultValue := stag.Get("doc")
+				if defaultValue != ""{
+					r, _ := regexp.Compile(`default\((.*)\)`)
+					if r.MatchString(defaultValue) {
+						res := r.FindStringSubmatch(defaultValue)
+						mp.Default = res[1]
+						switch realType{
+							case "int","int64", "int32", "int16", "int8":
+								mp.Default, _ = strconv.Atoi(res[1])
```

Comment:
I think we need to handler the error here and follow. As if the real type is int while the default value is not int. Then we need to panic it and tell the user which is wrong.